README.md

```diff
@@ -2,20 +2,16 @@
 
 ## Introduction
 
-The Islandora Bookmark module allows users to build, manage, share and track lists of objects. Manage your bookmarks under /islandora-bookmark. There are two ways to add bookmarks to your lists:
+The Islandora Bookmark module allows Drupal users to build, manage, share, track via RSS, and export lists of objects. A user's lists are managed at /islandora-bookmark, and objects can be added to lists individually or from a Solr search results page.
 
-1. Through object pages: every object page will have a little tool at the bottom to bookmark the
-current object.
-2. Through Solr searches: Islandora Bookmark provides a SOLR display with functionality to
-bookmark objects based on a custom Solr query.
 
 ## Requirements
 
 This module requires the following modules/libraries:
 
 * [Islandora](https://github.com/islandora/islandora)
-* [Tuque](https://github.com/islandora/tuque)
-* [Islandora Solr](https://github.com/islandora/islandora_solr_search) (optional)
+* [Islandora Solr](https://github.com/islandora/islandora_solr_search) (optional, provides functionality to bulk add objects)
```

Comment:
Hi, since neither Solr or Bibliography are real dependencies, should we use put them under some type of Optionals title? Or under "Islandora recommends these modules for ..."

Reply:
Maybe? Should we distinguish between how this module includes a lot of code that only works if you have solr installed (I'd say Solr provides the more useful interface to Bookmarks) vs how Bibliography is just one way to style the exports?

Reply:
+1 Adding  more information about why we 'recommend' each one seems like it can only help.